server/entities/classifier.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC
```

Comment:
What is abc?

Reply:
I read that it is required for abstract classes.

Reply:
Seems like this is an example import